.npmrc

```diff
@@ -0,0 +1,3 @@
+dedupe-direct-deps=true
+fund=false
+audit=false
```

Comment:
And this saves another second. Node package installation at 4 seconds should be safe to ignore now.